extensions/README.md

```diff
@@ -140,3 +130,74 @@ Since it's added to the beta array, you need to load the beta blocks as explaine
 
 `wp jetpack scaffold block "Jukebox" --keywords="music, audio, media"`
 
+## Good to know when developing Gutenberg extensions
+
+## The Build
+
+- Compiled extensions are output to `_inc/blocks`
+- You can view the various build commands in `package.json`
+- You can see the build configuration in `webpack.config.extensions.js`
+
+If you need to modify the build process, bear in mind that config files are also
+synced to WordPress.com via Fusion. Consult with a Jetpack crew member to ensure
+you test the new build in both environments.
+
+## Debugging
+
+Setting these might be useful for debugging with block editor:
+
+```php
+define( 'SCRIPT_DEBUG', true );
+define( 'GUTENBERG_DEVELOPMENT_MODE', true );
+```
+
+You could add these to `docker/wordpress/wp-config.php` in your Docker environment.
+
+[G Debugger](https://wordpress.org/plugins/g-debugger/) plugin might come handy, too.
+
+### Don't worry about dependencies
+
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalised and automatically enqueued in PHP for your extension.
+
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
+
+### Jetpack sidebar
+
+Jetpack sidebar is always registered in the editor and filled using [Slot Fill](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/slot-fill#readme) mechanism.
+
+Just render with `JetpackPluginSidebar` component from anywhere in your plugin's code:
+
+```jsx
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
+
+<JetpackPluginSidebar>
+	<PanelBody title={ __( 'My sidebar section', 'jetpack' ) }>
+		<p>Jetpack is Bestpack!</p>
+	</PanelBody>
+</JetpackPluginSidebar>
+```
+
+The sidebar won't show up at all if nothing isn't rendering in the sidebar's "slot".
+
+Remember to be mindful of posts, pages, custom post types and re-usable block post type. See Publicize and Shortlinks for examples how to limit functionality only to some specific post types or posts.
+
+### i18n
+
+As of 04/2019, `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
+
+Not possible:
+
+> Still confused? Check out <a>documentation</a> for more!
+
+Possible:
+
+> Still confused? <a>Check out documentation for more!</a>
+
+### Colours
```

Comment:
We should probably stick to american spellings (e.g. color) when possible, here and downer on the page.

Reply:
Hah, you.. can't... make ... me do i...

done in https://github.com/Automattic/jetpack/pull/12050/commits/1dd2f0aceb9f3ac8f5125261708a9cc79ce700ca